diff --git a/node_modules/rdf-parser-jsonld/index.js b/node_modules/rdf-parser-jsonld/index.js
index cbd5105..969b80d 100644
--- a/node_modules/rdf-parser-jsonld/index.js
+++ b/node_modules/rdf-parser-jsonld/index.js
@@ -12,22 +12,16 @@ var JsonLdParser = function (options) {
     options.importPrefixMap = true
   }
 
-  var expandAndFlatten = function (data, base, callback) {
-    jsonld.expand(data, {'base': base}, function (error, expanded) {
-      if (error) {
-        return callback(error)
-      }
-
-      jsonld.flatten(expanded, {}, function (error, flattened) {
-        if (error) {
-          return callback(error)
-        }
+  var expandAndFlatten = async function (data, base, callback) {
+    jsonld.expand(data, {'base': base}).then(function (expanded) {
+      jsonld.flatten(expanded, {}).then(function (flattened) {
 
         if (!('@graph' in flattened)) {
           return callback(new Error('no @graph property in flattened JSON-LD'))
         }
-
         callback(null, flattened['@graph'])
+      }).catch((error) => {
+          return callback(error)
       })
     })
   }
